Src/AutoFixtureUnitTest/AbstractRecursionIssue/Repro.cs

```diff
@@ -65,7 +65,7 @@ public void Workaround()
                     typeof(ItemBase),
                     typeof(FunkyItem)));
 
-            Assert.DoesNotThrow(() => fixture.Create<FunkyItem>());
+            AssertEx.DoesNotThrow(() => fixture.Create<FunkyItem>());
```

Comment:
Yep, agree. Replaced with what you suggested.
I created a standalone commit rather than rebase to not mess up the existing commits (and links to them).